test/util.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * All tests that tests any utility.
+ * Prevent any breaking of functionality
+ */
+import assert from 'assert';
+import typeOf from '../src/lib/util/typeOf';
+
+describe('Util', () => {
+  it('should validate different typeOf', () => {
+    assert.strictEqual(typeOf([]), 'array');
+    assert.strictEqual(typeOf(null), 'null');
+    assert.strictEqual(typeOf({}), 'object');
+    assert.strictEqual(typeOf(new Date()), 'date');
+    assert.strictEqual(typeOf('ezkemboi'), 'string');
```

Comment:
BTW, could you add one more test for `String('kemboi')` ?

Reply:
I will add

Reply:
Added the test @profnandaa